banman/util.go

```diff
@@ -0,0 +1,51 @@
+package banman
+
+import (
+	"net"
+)
+
+var (
+	// defaultIPv4Mask is the default IPv4 mask used when parsing IP
+	// networks from an address. This ensures that the IP network only
+	// contains the specified address.
+	defaultIPv4Mask = net.IPv4Mask(0xff, 0xff, 0xff, 0xff)
```

Comment:
Not a blocker, but IMO, CIDR notation is easier to follow. So this would be a `/32` in that notation .

Reply:
See https://golang.org/pkg/net/#CIDRMask

Reply:
Done using `net.CIDRMask`.